### PR TITLE
comments, css, js files

### PR DESCRIPTION
**Pull request by JurateJ**

### CODE REVIEW

**Comment by JurateJ** [comment]
Atiduodu cenzūrai

File: index.html [hunk 1]
```diff
@@ -11,6 +11,17 @@
 <body>
     <!-- header -->
     <!-- hero -->
+    <!-- creativity -->
+    <!-- intelligence -->
+    <!-- achievements -->
+    <!-- servicies -->
+    <!-- projects -->
+    <!-- updates -->
```
Comment (EgleJuske) [inline index.html:19]: between updates and works shoul be "clients" section, the same will go to css and js comments

**Review by EgleJuske — CHANGES_REQUESTED** [review]
mistake in file name "js/components/contact/renderContact,js"
should be  .js 

I don't know for sure, but maybe we have to change projects.js and creativity.js to renderProjects.js and renderCreativity.js

File: index.html (hunk 1 above)
Comment (JurateJ) [reply index.html:19]: Done

**Review by EgleJuske — DISMISSED** [review]
👌👌👌

**Review by Ramazinas — DISMISSED** [review]
Good start

**Review by Jjurgita — CHANGES_REQUESTED** [review]
One observation: better to use the word "services" not "servicies"

**Review by Jjurgita — APPROVED** [review]
Good to go. Thank you

**Review by EgleJuske — APPROVED** [review]
better every time 😊

**Review by Ramazinas — APPROVED** [review]
All ok

**Pull request merged**